velox/connectors/hive/HiveDataSink.cpp

```diff
@@ -667,6 +667,11 @@ uint32_t HiveDataSink::appendWriter(const HiveWriterId& id) {
   const auto* connectorSessionProperties =
       connectorQueryCtx_->sessionProperties();
   options.schema = getNonPartitionTypes(dataChannels_, inputType_);
+  auto sessionCfgParqVersion = connectorSessionProperties->get("parquet_writer_version");
```

Comment:
This should probably be added to the hiveConfig. The session parameter is associated with it `hive.parquet_writer_version` and is a session parm. Many hive parameters are both configurable in the catalog file as well as via session properties.
Take a look at the HiveConfig.